pkg/manager/collectors/harvester.go

```diff
@@ -0,0 +1,84 @@
+package collectors
+
+import (
+	"path/filepath"
+
+	"github.com/Jeffail/gabs/v2"
+	"github.com/sirupsen/logrus"
+)
+
+type harvesterModule struct {
+	c    *common
+	name string
+}
+
+func NewHarvesterModule(common *common, name string) *harvesterModule {
+	return &harvesterModule{
+		c:    common,
+		name: name,
+	}
+}
+
+func (module harvesterModule) generateYAMLs() {
+	logrus.Infof("[%s] generate YAMLs, yamlsDir: %s", module.name, module.c.yamlsDir)
+
+	extraResources := getHarvesterExtraResource()
+	for namespace, resourceLists := range extraResources {
+		dir := filepath.Join(module.c.yamlsDir, "namespaced", namespace)
+		objs, err := module.c.discovery.SpecificResourcesForNamespace(module.toObj, module.name, namespace, resourceLists, module.c.errorLog)
+
+		if err != nil {
+			logrus.Error("Unable to fetch namespaced resources")
+			return
+		}
+
+		for name, obj := range objs {
+			file := filepath.Join(dir, name+".yaml")
+			logrus.Debugf("Prepare to encode to yaml file path: %s", file)
+			module.c.encodeFunc(obj, file, module.c.errorLog)
+		}
+	}
+}
+
+func (module harvesterModule) toObj(b []byte, groupVersion, kind string, resources ...string) (interface{}, error) {
+	jsonParsed, err := module.c.toObjCommon(b, groupVersion, kind)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(resources) > 1 {
+		logrus.Warnf("Should not have more than one resources, resource: %v", resources)
```

Comment:
Suggest removing this since it will get logged later in the default case.

Reply:
Fixed. Please check [force-pushed](https://github.com/rancher/support-bundle-kit/compare/df5bbd80f07dde4e82cf3a6d7862cdba0406ebfd..61d1cd3f342fc550c5f2447b28e855b80f246e8e)